test/functional/p2p_blockfilters.py

```diff
@@ -21,6 +22,7 @@
 from test_framework.p2p import P2PInterface
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
+    assert_not_equal,
```

Comment:
Importing twice? (from `util` and `messages`)

Reply:
hmm yea I was trying to use this command to find all the places but it seems to be a bit more complex than I initially thought
```
git grep -l "assert .*!=" -- ':(exclude)*script.py' ./test/functional | xargs sed -i -e "/assert .*!=/ s/assert /assert_not_equal(/g" -e "/assert_not_equal/ s/ !=/,/g" -e "/assert_not_equal/ s/$/)/g" -e "s/test_framework.util import (/test_framework.util import (\n    assert_not_equal,/g" -e "s/import assert_equal/(\n    assert_equal,\n    assert_not_equal,\n)/g"
```

I might just manually add the imports and use the script to look for the `assert ... !=` as suggested by @Empact